vyper/codegen/module.py

```diff
@@ -100,6 +99,10 @@ def _runtime_ir(runtime_functions, all_sigs, global_ctx):
     internal_functions = [f for f in runtime_functions if _is_internal(f)]
 
     external_functions = [f for f in runtime_functions if not _is_internal(f)]
+
+    # returns True if any function's selector has trailing 0s
```

Comment:
two things -- this isn't a function so it doesn't return anything, also the code is not checking for trailing 0s, it's checking for exact equality to 0
```suggestion
    # check if any selector is 0
```

Reply:
thanks! implemented